packages/components/src/contentContainer/ContentContainer.js

```diff
@@ -0,0 +1,54 @@
+// @flow
+
+import * as React from 'react';
+import { View, ScrollView } from 'react-native';
+import { StyleSheet, type StylePropType } from '@kiwicom/universal-components';
+import { defaultTokens } from '@kiwicom/orbit-design-tokens';
+import { margaritaTokens } from '@kiwicom/margarita-design-tokens';
+
+type Props = {|
+  +children: React.Node | React.ChildrenArray<React.Node>,
+  +style?: StylePropType,
+  +scroll?: boolean,
```

Comment:
what about `isScroll` or something similar wich be on first look clear that take boolean value

Reply:
It's definitely better! Done